src/circuit/gadget/add_chip.rs

```diff
@@ -1,3 +1,5 @@
+//! `Add` chip implemetation.
```

Comment:
@str4d ([Link](https://github.com/zcash/orchard/pull/471#discussion_r2550151848)): ```suggestion
//! `Add` chip implementation.
```